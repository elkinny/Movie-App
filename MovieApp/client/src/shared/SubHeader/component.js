import React from 'react';

import styles from './style.scss';

const SubHeader = props => {
  return <div className={styles.subheader}>{props.children}</div>;
};

export default SubHeader;
