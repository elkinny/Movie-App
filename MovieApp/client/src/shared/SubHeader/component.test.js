import SubHeader from './component';
import React from 'react';
import { mount } from 'enzyme';

describe('SubHeader: ', () => {
  it('SubHeader to match snapshot', () => {
    const component = mount(<SubHeader>Hi</SubHeader>);
    expect(component).toMatchSnapshot();
  });
});
