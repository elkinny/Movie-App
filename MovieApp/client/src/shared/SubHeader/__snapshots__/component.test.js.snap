// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SubHeader:  SubHeader to match snapshot 1`] = `
<SubHeader>
  <div
    className="subheader"
  >
    Hi
  </div>
</SubHeader>
`;
